DOOMBFA.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+if [[ " $@ " =~ " -open-launcher " ]]
+then
+   mkdir -p "$HOME/.doombfa/base/lib"
```

Comment:
Does this really need to be relative to $HOME? Or you can put it into some XDG data dir/or some other dir inside the sandbox?

Reply:
Yes, it is. That path have to be accessible to the user in order to install an extension (not distributed with the Project due to licensing issues)